packages/aragon-wrapper/src/index.js

```diff
@@ -69,7 +69,10 @@ export const setupTemplates = (
 export default class Aragon {
   constructor (daoAddress, options = {}) {
     const defaultOptions = {
-      provider: detectProvider()
+      provider: detectProvider(),
+      defaultGasPriceFn: () => {
+        return this.web3.utils.toWei('20', 'gwei')
```

Comment:
I think the default function return a Promise even if we resolve immediately: `Promise.resolve(...)`

Reply:
@izqui can you elaborate on why this is necessary? The function is synchronous by nature and `await` consumes synchronous functions as well.

I will add it if that's what you want, but I don't see the value it in right now :)

Reply:
The reason I commented that was to make it more explicit to someone reading the code that the function was asynchronous, but of course this would be better solved with proper documentation :)